core/templates/pages/learner-dashboard-page/learner-dashboard-page.constants.ts

```diff
@@ -22,12 +22,18 @@ export const LearnerDashboardPageConstants = {
     COMPLETED: 'I18N_LEARNER_DASHBOARD_COMPLETED_SECTION',
     SUBSCRIPTIONS: 'I18N_LEARNER_DASHBOARD_SUBSCRIPTIONS_SECTION',
     FEEDBACK: 'I18N_LEARNER_DASHBOARD_FEEDBACK_SECTION',
-    PLAYLIST: 'I18N_LEARNER_DASHBOARD_PLAYLIST_SECTION'
+    PLAYLIST: 'I18N_LEARNER_DASHBOARD_PLAYLIST_SECTION',
+    GOALS: 'I18N_LEARNER_DASHBOARD_GOALS_SECTION',
```

Comment:
What is this section? Isn't it just current goals?

Reply:
Removed.